Jenkinsfile

```diff
@@ -51,6 +56,20 @@ node('rhel8'){
             sh "sftp -C ${UPLOAD_LOCATION}/snapshots/vscode-apache-camel/ <<< \$'put -p -r ${tgzFilesToPush[0].path}'"
 		}
     }
+
+	stage('Generate SBOM'){
+		packageVersion = sh(script: 'jq -rcM .version < package.json', returnStdout: true ).trim()
+		sh "cyclonedx-npm --omit dev --output-file node-sbom.json"
+		sh """./cyclonedx-linux-x64 merge \
+		--hierarchical \
+		--group com.github.camel-tooling \
+		--name vscode-apache-camel \
+		--version ${packageVersion} \
+		--input-files node-sbom.json camel-ls-sbom.json \
+		--output-file manifest.json
+		"""
+		archiveArtifacts artifacts:"**/camel-ls-sbom.json, **/manifest.json"

```

Comment:
I think we can use `manifest.json` directly, it will avoid to search through the whole directory